skia/RSkSVGNode.cpp

```diff
@@ -339,39 +291,66 @@ bool RSkSVGNode::setDashArrayAttribute(SkSVGAttribute attr, const std::vector<st
 bool RSkSVGNode::setTransformAttribute(SkSVGAttribute attr,const std::vector<Float> matrix) {
 
   if(matrix.size() == 6) {
-    RNS_LOG_DEBUG("\n"<<
+
+#ifdef ENABLE_RSKSVG_PROPS_DEBUG
+    RNS_LOG_INFO("\n"<<
                   " Matrix 0 : "<<matrix[0] << "\n" <<
                   " Matrix 1 : "<<matrix[1] << "\n" <<
                   " Matrix 2 : "<<matrix[2] << "\n" <<
                   " Matrix 3 : "<<matrix[3] << "\n" <<
                   " Matrix 4 : "<<matrix[4] << "\n" <<
                   " Matrix 5 : "<<matrix[5]);
+#endif/*ENABLE_RSKSVG_PROPS_DEBUG*/
 
-    SkMatrix svgTransforMatrix=SkMatrix::Translate(matrix[4],matrix[5]);
-    svgTransforMatrix.preConcat(SkMatrix::Scale(matrix[0],matrix[3]));
-    setAttribute(attr,SkSVGTransformValue(svgTransforMatrix));
+    SkMatrix svgTransforMatrix;
+    // Converting received Matrix from React Native framework which is in column major Order to row Major Order for SKia
```

Comment:
Can this done with any copy constructor

Reply:
SkMatrix doesn't have copy constructor, which takes column major & converts to row major